mava/components/jax/building/adders.py

```diff
@@ -264,7 +286,7 @@ def on_building_executor_adder(self, builder: SystemBuilder) -> None:
 
         adder = reverb_adders.ParallelSequenceAdder(
```

Comment:
according to the diagram, this component has a dependency on GAE so should add a required components method (the other 2 deps are covered by the base class)